www/showtags

```diff
@@ -99,10 +97,24 @@ function calcDistBin($val, $bins)
 if (count($tags) == 0) {
     echo "<i>There are currently no tags in the database</i>";
 } else {
+    $datalist = get_req_data("datalist");
 
     $cloud = get_req_data("cloud");
 
-    if ($cloud) {
+    if ($datalist) {
+        header('Content-Type: text/xml');
+        header('Cache-Control: public, max-age=3600');
+        echo "<datalist id=\"tags-list\">\n";
+        foreach ($tags as $t) {
+            $freq = $t['freq'];
+            if ($freq < 10) continue;
```

Comment:
this check can be part of the SQL above. In the test DB it will fetch 861 records instead of 4937.

It can be simple `having freq >= ?` and then the query param is just `$datalist ? 10 : 0`

Reply:
done!